File: 10209 Is This Integration ?/solution.c
#include <stdio.h>
#include <math.h>

int main() {
    double PI = acos(-1), a;
    while(scanf("%lf", &a) == 1)
        printf("%.3f %.3f %.3f\n", a * a * (PI / 3 + 1 - sqrt(3)), 4 * a * a * (PI / 12 - 1 + sqrt(3) / 2), 4 * a * a * (-PI / 6 + 1 - sqrt(3) / 4));
    return 0;
}

File: 10209 Is This Integration ?/solution.java
import java.io.*;

public class Main {
	public static void main(String[] args) throws IOException {
        StreamTokenizer st = new StreamTokenizer(System.in);
        StringBuilder output = new StringBuilder();
        while(st.nextToken() == StreamTokenizer.TT_NUMBER) {
            double a = st.nval;
            output.append(String.format("%.3f %.3f %.3f\n", a * a * (Math.PI / 3 + 1 - Math.sqrt(3)), 4 * a * a * (Math.PI / 12 - 1 + Math.sqrt(3) / 2), 4 * a * a * (-Math.PI / 6 + 1 - Math.sqrt(3) / 4)));
        }
        System.out.print(output);
	}
}
